easybuild/easyconfigs/b/Bader/Bader-1.03-intel-2018b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.03'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{'download_filename': 'bader.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['3de8af94bd64c43812de7ebad3e9209cc5e29e6732679fcf88f72a1b382bc81a']
+
+skipsteps = ['configure']
```

Comment:
There is no configure step to skip with MakeCp, you can however turn it on by using "with_configure = True"

Reply:
+1, please drop